stdlib/runtime/gc.gr

```diff
@@ -165,7 +171,8 @@ export let incRef = (userPtr: WasmI32) => {
 }
 
 let rec decRef = (userPtr: WasmI32, ignoreZeros: Bool) => {
-  if (WasmI32.eqz(userPtr & Tags._GRAIN_GENERIC_TAG_MASK) && WasmI32.ne(userPtr, 0n)) {
+  if (WasmI32.eqz(userPtr & Tags._GRAIN_GENERIC_TAG_MASK) &&
+    WasmI32.ne(userPtr, 0n)) {
```

Comment:
Ugh, don't like these wraps

Reply:
I believe I have to wrap there for the new parser